src/index.html

```diff
@@ -17,7 +17,6 @@
         
     </section>
 
-</body>
-<!-- <script type="module" src="./index.js"></script> -->
```

Comment:
Although I am highlighting this piece of code, the changes I am about to request are not related to it. I could not highlight the relevant piece of code in the Files Changes section of your pull request. Kindly make the following changes
- [ ] Kindly remove the Microverse logo from your README.md file. If you have your logo, you can use it but if you don't, kindly remove it. 👍 💻 
- [ ] Kindly remove appreciation for Cindy Shin in the Acknowledgement Section of your README.md file. I guess that was used during the HTML/CSS/JavaScript capstone project. 👍 😄 
- [ ] Kindly remove all unused/commented codes (Line 7, Line 16, Line 43 - 46) in your _**webapack.config.js**_ file. Removing such makes your code look clean.
- [ ]  Kindly remove the unused/commented code (Line 24) in your _**taskData.js**_ file. Removing such makes your code look clean.